web/src/main/java/no/nav/modiapersonoversikt/rest/kontaktinformasjon/KontaktinformasjonController.kt

```diff
@@ -1,55 +1,74 @@
 package no.nav.modiapersonoversikt.rest.kontaktinformasjon
 
-import no.nav.modiapersonoversikt.consumer.dkif.consumer.DkifService
 import no.nav.modiapersonoversikt.infrastructure.naudit.Audit
 import no.nav.modiapersonoversikt.infrastructure.naudit.Audit.Action.*
 import no.nav.modiapersonoversikt.infrastructure.naudit.AuditIdentifier
 import no.nav.modiapersonoversikt.infrastructure.naudit.AuditResources.Person
+import no.nav.modiapersonoversikt.infrastructure.scientist.Scientist
 import no.nav.modiapersonoversikt.infrastructure.tilgangskontroll.Policies
 import no.nav.modiapersonoversikt.infrastructure.tilgangskontroll.Tilgangskontroll
-import no.nav.tjeneste.virksomhet.digitalkontaktinformasjon.v1.meldinger.WSHentDigitalKontaktinformasjonResponse
+import no.nav.modiapersonoversikt.service.dkif.Dkif
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/rest/person/{fnr}/kontaktinformasjon")
-class KontaktinformasjonController @Autowired constructor(private val dkifService: DkifService, private val tilgangskontroll: Tilgangskontroll) {
+class KontaktinformasjonController @Autowired constructor(
+    @Qualifier("DkifSoap") private val dkifSoapService: Dkif.Service,
+    @Qualifier("DkifRest") private val dkifRestService: Dkif.Service,
+    private val tilgangskontroll: Tilgangskontroll
+) {
+
+    val dkifExperiment = Scientist.createExperiment<Dkif.DigitalKontaktinformasjon>(
+        Scientist.Config(
+            name = "dkif",
+            experimentRate = 0.1
+        )
+    )
 
```

Comment:
Ville kanskje ha bruk Unleash som alternativ for kven som får nye og gamle, gjer vår verifikasjon litt lettare i testmiljø.

Reply:
Dette ble diskutert på standup, og etter [1aced56](https://github.com/navikt/modiapersonoversikt-api/pull/407/commits/1aced5679e15783cfd33a035ff29d67d0fc8bfdd) vil verifikasjon være lettere da man kan tvinge eksperimentet til å kjøre ved en `?forceExperiment` suffix på kallet til endepunktet.